### DISCUSSION TITLE
Comments on the road to v4!

### DISCUSSION THREAD

**Discussion by cyp3rius** [VirtusLab-Open-Source/strapi-plugin-comments#92] [Ideas]
Hello Folks!

As the non-beta v4 of Strapi has been recently released, we've initialised our work on rewriting (yep, not migrating, but rewriting) the whole Comments plugin to the new version. 

I would like to head from you:
- what do you want to see on the "dashboard" / first page
- was the UI and tree structure of comments intuitive enough
- what was missing?

**Comment by affinage-digital** [comment]
Do you know when the first release will be?

_I have not used the plugin before, so I cannot answer your questions_

**Reply by cyp3rius** [reply]
Expecting New Year eve :)

**Comment by Tragio** [comment]
@cyp3rius really curious to see the new version. 🚀  From my side, I would love to have the author's thumbnail included in the Get Comments or maybe written in the docs what is the best way to include it there.

**Reply by cyp3rius** [reply]
@Tragio the goal for v4 is to replicate all current v3 functionalities for RC version. In the early beta there won't be (yet):
- GQL support
- Migration script

All additions are planned in next versions :)

**Comment by cyp3rius** [comment]
Because of just a couple "glitches" we decided to publish the first beta early next week officially via npm, but now you're able to get the latest code on [feat/strapi-v4 ](https://github.com/VirtusLab-Open-Source/strapi-plugin-comments/tree/feat/strapi-v4) branch.

Happy New Year !

**Reply by du0ngtrunghieu** [reply]
It's worth looking forward to the next version, thank you team

**Comment by cyp3rius** [comment]
`v2.0.0-beta.1` has been just released :) Read the [release notes](https://github.com/VirtusLab-Open-Source/strapi-plugin-comments/releases/tag/v2.0.0-beta.1) please!
